grades/management/commands/populate_combined_final_grades.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management import BaseCommand
+
+from courses.models import Course
+from grades.api import update_or_create_combined_final_grade
+from grades.models import ProctoredExamGrade
+from micromasters.utils import now_in_utc
+
+
+class Command(BaseCommand):
+    """
+    Finds all passing exam grades for courses that have frozen runs
+    """
+    help = "Finds all users that might need a CombinedFinalGrade"
+
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
+        courses = Course.objects.filter(
+            program__live=True,
+            program__financial_aid_availability=True,
+            has_exam=True
```

Comment:
`has_exam` is a `@property` on the `Course` class, so this query ends up failing. We don't have a ton of courses so it may just be simpler to remove this and then filter to `course.has_exam` in code.